Makefile

```diff
@@ -104,6 +118,9 @@ clean: ## Clean up your working environment
 really-clean: clean cleanup-ci ## Really clean up the working environment
 	@rm -f $(KUBERNETES_FILES)
 
+clean-debug: ## Revert changes made to support debug image
```

Comment:
This is a big IMO, so other reviewers can downvote me if I'm wrong. I don't believe that we need this make target (or the associated script), with the move to a broker-apb. My reason for this is that, with our move to a broker-apb, you could simply deprovision the broker and start fresh. I also normally view `clean` make targets to be cleaning up stuff on the host...I'm not super concerned about the cluster.

Reply:
I'm fine with that, really just wanted this t get state back to where it was in case there were any concerns in that regard.